Documentation/ceph-cluster-crd.md

```diff
@@ -1489,3 +1490,31 @@ data:
 
 Note: if you are using self-signed certificates (not known/approved by a proper CA) you must pass `VAULT_SKIP_VERIFY: true`.
 Communications will remain encrypted but the validity of the certificate will not be verified.
+
+##### Ceph CSI configuration
+
+Ceph-CSI is capable of encrypting PVs requests.
+Rook can configure Ceph-CSI to turn on encryption with Key Management System support.
```

Comment:
Ceph-CSI is going to support a 3rd way for encrypting PVs now, aptly named "Vault Tokens". Is there any interest or intention to support encryption based on Vault access by Kubernetes ServiceAccount or the no-KMS (password from a Secret)? Depending on those, the contents /(in JSON) of the generated ConfigMap would change a little.

Reply:
I believe the intent, for now, is to stick with anything that relates to the KMS environment but if we can easily incorporate more scenarios I'm fine with that too.